.git-hooks/install-pull-update-deps

```diff
@@ -1,8 +1,37 @@
 #!/usr/bin/env bash
 # MIT © Minh Duc Vo <minhducsun2002@gmail.com>
 
+# Copyright (C) 2019 Trung Nguyen (TrungNguyen1909)
+
+# Zero Clause BSD license {{{
+#
+# Permission to use, copy, modify, and/or distribute this software for any purpose with or without fee is hereby granted.
+#
+# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+# }}}
+realpath() {
+    ourpwd=$PWD
+    cd "$(dirname "$1")"
+    link=$(readlink "$(basename "$1")")
+    status=$?
+    while [ $status -eq 0 ]; do
+        cd "$(dirname "$link")"
+        t=$(readlink "$(basename "$link")")
+        status=$?
+        if [ $status -eq 0 ]
+        then
+            link=$t
+        fi
+    done
+    realpath="$PWD/$(basename "$link")"
+    cd "$ourpwd"
+    echo "$realpath"
+}
+
+# ===========
+
```

Comment:
This seems like a lot of code to keep in the repo for such a narrow use case. I think a better approach would be to just document how to install the git hook manually by copying it to the correct location, without including a script to install it.